packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -30,13 +30,13 @@ import { ContractId } from '@hashgraph/sdk';
 
 // Assertions and constants from local resources
 import Assertions from '../helpers/assertions';
-import RelayAssertions from '../../../../packages/relay/tests/assertions';
+import RelayAssertions from '@hashgraph/json-rpc-relay/tests/assertions';
```

Comment:
We keep going back and forth on this standard ever 6 months.
I believe using this identifier means you have to `npm run build` to get local changes to get picked up.
Other than avoiding the need to change the path on file moves does this have other benefits?

Reply:
Tbh, I don't really have a strong opinion here. 

I slightly like this approach more because we can use the declared version for the `@hashgraph/json-rpc-relay` dependency as a source of truth and we are decoupling the `server` package from the `relay` package. This allows us to completely separate the packages into their own independent modules instead of relying on files that are outside of the packages' scope.

Reply:
I agree.
I like the decoupled approach better.
Can you open a ticket for this also to address later?

Reply:
Done, https://github.com/hashgraph/hedera-json-rpc-relay/issues/2685